### PR TITLE
Excel like incell edit

### PR DESCRIPTION
**Pull request by marin-bratanov**

### CODE REVIEW

**Comment by marin-bratanov** [comment]
This also includes reviewing the demo updates around kbd nav

**Review by ntacheva — APPROVED** [review]

File: components/treelist/editing/incell.md [hunk 1]
```diff
@@ -10,23 +10,27 @@ position: 4
...
+To enable InCell editing mode, set the `EditMode` property of the treelist to `Telerik.Blazor.TreeListEditMode.Incell`. You can handle the `OnUpdate`, `OnCreate` and `OnDelete` events to perform the CUD operations, as shown in the example below.
 
-To enable InCell editing mode, set the `EditMode` property of the treelist to `Telerik.Blazor.TreeListEditMode.Incell`, then handle the CRUD events as shown in the example below.
+To add a new item, you must also add a [toolbar]({%slug treelist-toolbar%}) with an `Add` command. `OnCreate` will fire immediately when you click the `Add` button, see the [Notes](#notes) below.
 
+The `OnUpdate` event always fires for the last edited cell on the row - when you remove focus from the grid, or when you press `Enter` to go to the next row.
```
Comment (ntacheva) [inline components/treelist/editing/incell.md:28]: should be treelist not grid

Comment (marin-bratanov) [reply components/treelist/editing/incell.md:28]: Yes, please fix it :)

Comment (ntacheva) [reply components/treelist/editing/incell.md]: Done

**Pull request merged**